cmd/crc/cmd/oc_env.go

```diff
@@ -38,10 +38,12 @@ func runOcEnv(args []string) error {
 	}
 	proxyConfig := consoleResult.ClusterConfig.ProxyConfig
 	fmt.Println(shell.GetPathEnvString(userShell, constants.CrcOcBinDir))
-	if proxyConfig.IsEnabled() {
-		fmt.Println(shell.GetEnvString(userShell, "HTTP_PROXY", proxyConfig.HTTPProxy))
-		fmt.Println(shell.GetEnvString(userShell, "HTTPS_PROXY", proxyConfig.HTTPSProxy))
-		fmt.Println(shell.GetEnvString(userShell, "NO_PROXY", proxyConfig.GetNoProxyString()))
+	if proxyConfig != nil {
+		if proxyConfig.IsEnabled() {
+			fmt.Println(shell.GetEnvString(userShell, "HTTP_PROXY", proxyConfig.HTTPProxy))
+			fmt.Println(shell.GetEnvString(userShell, "HTTPS_PROXY", proxyConfig.HTTPSProxy))
+			fmt.Println(shell.GetEnvString(userShell, "NO_PROXY", proxyConfig.GetNoProxyString()))
+		}
```

Comment:
Shouldn't `crc oc-env` error out when the `podman` preset is in use?

Reply:
Yes,l. There is an issue for that from podman's perspective. Same issue